test/integration/annotation_spec.js

```diff
@@ -13,49 +13,72 @@
  * limitations under the License.
  */
 
+const { closeAll, loadAndWait } = require("./commons.js");
+
 describe("Annotation highlight", () => {
   describe("annotation-highlight.pdf", () => {
     let pages;
 
     beforeAll(async () => {
-      pages = await Promise.all(
-        global.integrationSessions.map(async session => {
-          const page = await session.browser.newPage();
-          await page.goto(
-            `${global.integrationBaseUrl}?file=/test/pdfs/annotation-highlight.pdf`
-          );
-          await page.bringToFront();
-          await page.waitForSelector("[data-annotation-id='19R']", {
-            timeout: 0,
-          });
-          return page;
-        })
+      pages = await loadAndWait(
+        "annotation-highlight.pdf",
+        "[data-annotation-id='19R']"
       );
     });
 
     afterAll(async () => {
-      await Promise.all(
-        pages.map(async page => {
-          await page.close();
-        })
-      );
+      await closeAll(pages);
     });
 
     it("must show a popup on mouseover", async () => {
       await Promise.all(
-        pages.map(async page => {
+        pages.map(async ([name, page]) => {
           let hidden = await page.$eval(
             "[data-annotation-id='21R']",
             el => el.hidden
           );
-          expect(hidden).toEqual(true);
+          expect(hidden).withContext(`In ${name}`).toEqual(true);
           await page.hover("[data-annotation-id='19R']");
           await page.waitForTimeout(100);
           hidden = await page.$eval(
             "[data-annotation-id='21R']",
             el => el.hidden
           );
-          expect(hidden).toEqual(false);
+          expect(hidden).withContext(`In ${name}`).toEqual(false);
+        })
+      );
+    });
+  });
+});
+
+describe("Checkbox annotation", () => {
+  describe("issue12706.pdf", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("issue12706.pdf", "[data-annotation-id='63R']");
+    });
+
+    afterAll(async () => {
+      await closeAll(pages);
+    });
+
+    it("must click on different checkboxes and they must behave like radio buttons", async () => {
+      const selectors = [63, 70, 79].map(n => `[data-annotation-id='${n}R']`);
+      await Promise.all(
+        pages.map(async ([name, page]) => {
+          for (const sel of selectors) {
```

Comment:
`sel` -> `selector` and `otherSel` -> `otherSelector` for readability in this function.